Sources/Xcore/Cocoa/Components/Currency/CurrencyFormatter/CurrencyFormatter.swift

```diff
@@ -21,6 +21,8 @@ public class CurrencyFormatter: Currency.SymbolsProvider {
         // design consistent (e.g., German (de) locale: "1.000,11 $" → "$1.000,11").
         $0.currencySymbol = currencySymbol
         $0.isDecimalEnabled = true
+        // When truncating fraction digits, if needed, we should round up.
```

Comment:
```suggestion
        // When truncating fraction digits, if needed, we should round up.
        // For example, `0.165` → `0.17` instead of `0.16`. 
```